config.yaml

```diff
@@ -153,49 +153,60 @@ commands:
         required: true
 templates:
   app_list:
-    slack: "{{name}} - {{owner_email}}"
-    hipchat: "{{name}} - {{owner_email}}"
+    body: |-
+      | Name | Owner |
+      | ---- | ----- |
+      ~each var=$results~
+      | ~$item.name~ | ~$item.owner_email~ |
+      ~end~
   app_info:
-    slack: |
-      Dynos: {{dynos}}
-      Git URL: {{git_url}}
-      Owner: {{owner_email}}
-      Region: {{region}}
-      Repo Size: {{repo_size}}
-      Slug Size: {{slug_size}}
-      Stack: {{stack}}
-      Web URL: {{web_url}}
-    hipchat: |
-      Dynos: {{dynos}}
-      Git URL: {{git_url}}
-      Owner: {{owner_email}}
-      Region: {{region}}
-      Repo Size: {{repo_size}}
-      Slug Size: {{slug_size}}
-      Stack: {{stack}}
-      Web URL: {{web_url}}
+    body: |-
```

Comment:
For what it's worth, I'd been doing the various "info" templates with a conditional on whether there is 1 or more-than-1 result. If 1, I was doing similar to this template, but bolding the labels. If more than 1, I generated a table instead.

Generally, the "info" commands should only be called where there will only be one result. Crazy folks might end up calling it in a loop (e.g., `pipewrench:for`), though, and then it'd degrade nicely.

Long term I think we can probably end up consolidating some of our "info" and "list" commands, such that they operate like this (a lot of the templates are identical.